_posts/2016-01-10-clojure-vim-environment.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+name: clojure-vim-environment
+title: 'Clojure VIM Environment'
+date: 2016-01-10 00:20:00 +00:00
+author: Mashooq Badar
+image:
+    src: /assets/img/custom/blog/vimenv.png
+tags:
+- clojure 
+- vim
+- craftsmanship
+---
+
+The [Cursive](https://cursive-ide.com/) plugin for Clojure is now out of its beta testing phase. I have been using it throughout its beta program and, I must say, it is pretty good. Even though it does not have the refactoring power of the IntelliJ for Java, but then Clojure is dynamic and asking for that kind of power is asking for the impossible. 
```

Comment:
The last sentence in this opening paragraph is not grammatically correct. Suggest re-wording as:

```"Sure, the refactoring capabilities are not quite on a par with those of IntelliJ for Java, but then Clojure is dynamic and asking for that kind of power is asking for the impossible".```

...or something similar.